addon/package.json

```diff
@@ -27,11 +39,15 @@
     "lint:js:fix": "eslint . --fix"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.26.10"
```

Comment:
we don't need it anymore?

Reply:
No, according to tutorial part 4 step 3 https://github.com/embroider-build/embroider/blob/main/PORTING-ADDONS-TO-V2.md#part-4-convert-addon-to-v2